okhttp/src/main/java/okhttp3/internal/platform/Android10Platform.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.platform;
+
+import android.annotation.SuppressLint;
+import android.net.ssl.SSLSockets;
+import java.util.List;
+import javax.annotation.Nullable;
+import javax.net.ssl.SSLParameters;
+import javax.net.ssl.SSLSocket;
+import okhttp3.Protocol;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
+
+/** Android 10+. */
+@SuppressLint("NewApi")
+class Android10Platform extends AndroidPlatform {
+  Android10Platform(Class<?> sslParametersClass) {
+    super(sslParametersClass, null, null, null, null);
```

Comment:
Main change from 3.14.x is here, we need to pass in diff set of OptionalMethod, instead of Method.

Reply:
null, null, null, null vs. null, null, null, null, null!